src/autoload.php

```diff
@@ -5,6 +5,12 @@
  * @license   https://github.com/laminas/laminas-zendframework-bridge/blob/master/LICENSE.md New BSD License
  */
 
+if (PHP_VERSION_ID >= 71000) {
+    require __DIR__ . '/autoload-functions-containerconfigtest.php';
+    require __DIR__ . '/autoload-functions-diactoros.php';
+    require __DIR__ . '/autoload-functions-stratigility.php';
```

Comment:
Hmm... Stratigility functions were first added in 2.2, which still supported 5.6. Which means I'll have to update #7 to likely do conditional requires as well.

Reply:
Yes, I've added it here for simplicity, as we need to add better support for all versions of these packages.